### DISCUSSION TITLE
Onion interface

### DISCUSSION THREAD

**Discussion by Linkinterteid** [OnionUI/Onion#1295] [General]
Why are there such weak options for customizing themes? A font with a shadow, white or light text looks terrible! Remove the shadow for a normal design! The list of games overlaps with covers, the favorites icon overlaps the game name, there is no text scrolling, why is the interface so terrible!? I’m not a programmer, I don’t know how to change this, but as a designer I took on the task of creating a theme, but when I saw all the horrors, I stopped wanting to make a theme.

Почему такие слабые возможности кастомизации тем оформления? Шрифт с тенью, белый или светлый текст смотрится ужасно! Уберите тень для нормального дизайна! Список игр перекрывается обложками, иконка избранного перекрывает название игры, нет прокрутки текста, ну почему интерфейс такой ужасный!? Я не программист я не знаю как это поменять, но как дизайнер я брался за создание темы, но когда увидел все ужасы, то мне перехотелось делать тему.

**Comment by Sichroteph** [comment]
Уважаемый Linkinterteid,

Ах, какое восхитительное сообщение, чтобы сделать наш день ярче! Ваши красноречивые выражения неудовлетворенности действительно тронули наши сердца, и мы не могли не поразиться художественному мастерству, с которым вы изобразили полотно своего недовольства. Это похоже на симфонию фрустрации, и мы чувствуем себя почетными быть вашей аудиторией.

Прежде всего, позвольте выразить нашу искреннюю благодарность за подвиг, который вы совершили, украшая наш скромный интерфейс вашим, несомненно, исключительным дизайном. Мы понимаем, что навигация по изменчивым водам нашего кодовой базы может быть сравнима с путешествием по лабиринту чудес и ужасов, и мы восхваляем вашу храбрость перед такими трудностями.

Теперь давайте рассмотрим ваши заботы с достоинством, которое они заслуживают:

Слабые опции для настройки тем:
Ах, танец теней и цветов текста! Мы должны признать, что наша милосердная группа разработчиков действительно рассматривала добавление функции чтения вашего ума и автоматической настройки темы в соответствии с вашим настроением, но увы, закрытые источники еще не догнали наши великие амбиции. Мы будем продолжать мечтать.

Список игр с наложенными обложками:
Это как шедевр сюрреализма, не так ли? Имитированное наложение - это приветствие хаотичной красоте жизни. Если бы только все мы могли бы оценить авангардный характер наших дизайнерских решений.

Иконка "Избранное" перекрывает название игры:
Ах, сладкий обхват наложения! Это метафора перекрывающихся сложностей существования. Но не беспокойтесь, мы уверены, что ваше художественное чувство сможет найти радость в гармонии хаоса.

Отсутствие прокрутки текста:
Прокрутка текста - это уже прошлый сезон, дорогая. Мы предпочитаем бросить вызов нашим пользователям в игре "найди скрытый текст". Это как современная охота за сокровищами, не так ли?

В заключение, ваши вдохновенные слова побудили нас создать новую категорию: "Критика как искусство". Мы с нетерпением ждем ваш шедевр темы, если вы решите снова одарить нас им. Пусть ваше художественное духовное развитие продолжается, и пусть вы находите радость в неожиданных тонкостях нашего интерфейса.

С ироническим восхищением,

Милосердные разработчики
